pages/index.test.js

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react';
+import Home from '.';
+
+describe(Home, () => {
+  it('renders a heading', () => {
+    render(<Home />);
+
+    const heading = screen.getByTestId('welcome-text');
+    expect(heading).toBeVisible();
```

Comment:
How about adding some space here to separate the declaration and assertion?💡 

```suggestion
    const heading = screen.getByTestId('welcome-text');

    expect(heading).toBeVisible();
```

Reply:
sure, done here https://github.com/tyrro/react-survey/pull/49 👍